.github/workflows/release-snap.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright The ORAS Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: release-snap
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'release version, like v1.2.0-beta.1'
+        required: true
+      isStable:
+        type: boolean
+        description: 'check for stable release'
+        default: false
+
+jobs:
+  release-snap:
+    strategy:
+      matrix:
+        arch: [amd64,arm64,s390x]
```

Comment:
This this array of string require quotes?

Reply:
Nice catch, updated